modules/dreamview/conf/hmi_modes/mkz_64beam.pb.txt

```diff
@@ -0,0 +1,160 @@
+cyber_modules {
+  key: "Camera"
+  value: {
+    dag_files: "/apollo/modules/drivers/camera/dag/camera.dag"
+    dag_files: "/apollo/modules/drivers/video/dag/video.dag"
+  }
+}
+cyber_modules {
+  key: "Canbus"
+  value: {
+    dag_files: "/apollo/modules/canbus/dag/canbus.dag"
+  }
+}
+cyber_modules {
+  key: "Control"
+  value: {
+    dag_files: "/apollo/modules/control/dag/control.dag"
+  }
+}
+cyber_modules {
+  key: "GPS"
+  value: {
+    dag_files: "/apollo/modules/drivers/gnss/dag/gnss.dag"
+  }
+}
+cyber_modules {
+  key: "Guardian"
+  value: {
+    dag_files: "/apollo/modules/guardian/dag/guardian.dag"
+  }
+}
+cyber_modules {
+  key: "Localization"
+  value: {
+    dag_files: "/apollo/modules/localization/dag/dag_streaming_msf_localization.dag"
+  }
+}
+cyber_modules {
+  key: "Perception"
+  value: {
+    dag_files: "/apollo/modules/perception/production/dag/dag_streaming_perception.dag"
+  }
+}
+cyber_modules {
+  key: "Traffic Light"
+  value: {
+    dag_files: "/apollo/modules/perception/production/dag/dag_streaming_perception_trafficlights.dag"
+  }
+}
+cyber_modules {
+  key: "Planning"
+  value: {
+    dag_files: "/apollo/modules/planning/dag/planning.dag"
+  }
+}
+cyber_modules {
+  key: "Storytelling"
+  value: {
+    dag_files: "/apollo/modules/storytelling/dag/storytelling.dag"
+  }
+}
+cyber_modules {
+  key: "Prediction"
+  value: {
+    dag_files: "/apollo/modules/prediction/dag/prediction.dag"
+  }
+}
+cyber_modules {
+  key: "Radar"
+  value: {
+    dag_files: "/apollo/modules/drivers/radar/conti_radar/dag/conti_radar.dag"
+  }
+}
+cyber_modules {
+  key: "Routing"
+  value: {
+    dag_files: "/apollo/modules/routing/dag/routing.dag"
+  }
+}
+cyber_modules {
+  key: "Transform"
+  value: {
+    dag_files: "/apollo/modules/transform/dag/static_transform.dag"
+  }
+}
+cyber_modules {
+  key: "Velodyne"
+  value: {
+    dag_files: "/apollo/modules/drivers/velodyne/dag/velodyne_64_16.dag"
+  }
+}
+cyber_modules {
```

Comment:
Do we need third_party_perception on? This probably not needed when Lidar 64 is on.

Reply:
OK, I remove it.